File: case01/java/Principal.java
public class Principal
{
    public static void main(String[] args){
        
        System.out.println("Cliente 1");
        Cliente c1 = new Cliente(Teclado.leString("Nome: "), Teclado.leInt("Idade: "), Teclado.leChar("Sexo: "), 
            Teclado.leString("email: " ), Teclado.leString("Senha: "), new Livro(Teclado.leString("Título: "), 
            Teclado.leString("Autor: "), Teclado.leInt("Ano de publicação: "), Teclado.leInt("Número de páginas: "), 
            Teclado.leDouble("Preço: ")));
    
        System.out.println("Cliente 2");
        Cliente c2 = new Cliente(Teclado.leString("Nome: "), Teclado.leInt("Idade: "), Teclado.leChar("Sexo: "), 
            Teclado.leString("email: " ), Teclado.leString("Senha: "), new Livro(Teclado.leString("Título: "), 
            Teclado.leString("Autor: "), Teclado.leInt("Ano de publicação: "), Teclado.leInt("Número de páginas: "), 
            Teclado.leDouble("Preço: ")));
            
        System.out.println("Cliente 3");
        Cliente c3 = new Cliente(Teclado.leString("Nome: "), Teclado.leInt("Idade: "), Teclado.leChar("Sexo: "), 
            Teclado.leString("email: " ), Teclado.leString("Senha: "), new Livro(Teclado.leString("Título: "), 
            Teclado.leString("Autor: "), Teclado.leInt("Ano de publicação: "), Teclado.leInt("Número de páginas: "), 
            Teclado.leDouble("Preço: ")));
        
        System.out.println(c1);
        System.out.println();
        System.out.println(c2);
        System.out.println();
        System.out.println(c3);
        System.out.println();
        
        c1.imprimeInformacoes();
        c2.imprimeInformacoes();
        c3.imprimeInformacoes();
    }
    
}

File: case01/python/Principal.py
from Cliente import Cliente
from Livro import Livro

if __name__ == "__main__":
    c1 = Cliente("Cliente_1", "@gmail","sd96234*",Livro("Livro_1", "Autor_1", 200, 58.99, 2023),"M",24)
    c2 = Cliente("Cliente_2", "@gmail","@jdk877s",Livro("Livro_2", "Autor_2", quantPag=150, preco=20.99),"F",21)
    c3 = Cliente(input("Nome do cliente: "), input("E-mail: "),input("Senha: "),Livro(input("Título: "), input("Autor: "), int(input("Número de páginas: ")), float(input("Preço: "))),int(input("Idade: ")),input("Sexo: "))
    
    print(c1)
    print(c2)
    print(c3)

